docs/CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ Version 3.42.0-eisop4 (April ?, 2024)
 
 **Implementation details:**
 
+Deprecated the `GenericAnnotatedTypeFactory#addComputedTypeAnnotations(Tree, AnnotatedTypeMirror, boolean)` overload.
+Instead, set `GenericAnnotatedTypeFactory#useFlow` appropriately and
+call `GenericAnnotatedTypeFactory#addComputedTypeAnnotations(Tree, AnnotatedTypeMirror)`.
+Add `GenericAnnotatedTypeFactory#addComputedTypeAnnotationsWithoutFlow(Tree, AnnotatedTypeMirror)` for the old behavior,
+subclasses can only call this method but not override it.
```

Comment:
Is this part important? How about more clearly describing which methods should be overridden now, because that changes from before.

Reply:
Updated the changelog.